server/src/app.ts

```diff
@@ -37,6 +37,11 @@ export abstract class AppBase {
     this.addStaticRoutes();
   }
 
+  startBackgroundProcess(): void {
+    // optional method that starts background processing.
```

Comment:
What do we mean by optional here -- optional for subclasses to extend it, or optional in that it is only called in certain circumstances? Worth clarifying comment and/or adding method doc. Also, does anything ever shut down background processing, or, how would one handle cleanup?

Reply:
Improved docs in the split off PR.  Thanks for the feedback!